packages/sourcecred/src/core/ledger/ledger.js

```diff
@@ -769,6 +793,55 @@ export class Ledger {
     account.payoutAddresses.delete(getCurrencyKey(currencyResult.value));
   }
 
+  /**
+   * Enabling accounting has the following effects on the ledger:
+   * - enables grain transfers
+   * - all accounts will start with a zero balance again, which will accumulate
+   *   normally from distributions
+   */
+  enableAccounting(): Ledger {
+    if (!this._balanceAccountingEnabled) {
+      this._createAndProcessEvent({
+        type: "ENABLE_ACCOUNTING",
+      });
+    }
+    // no-op if already enabled
+    return this;
+  }
+
+  _enableAccounting(_: EnableAccounting) {
+    this._balanceAccountingEnabled = true;
+  }
+
+  /**
+   * Disabling accounting has the following effects on the ledger:
+   * - disables grain transfers
+   * - zeroes out all balances
+   * - deactivates accounts that don't have a PayoutAddress set for the
+   *   current instance configuration, and prevents them from activating until
+   *   they do.
+   */
+  disableAccounting(): Ledger {
+    // ensure accounting isn't already disabled
```

Comment:
redundant comment, the code is plenty readable on its own + already have a no-op comment below